Source/Eto.Gtk/Forms/GtkControl.cs

```diff
@@ -378,6 +378,8 @@ protected override WeakConnector CreateConnector()
 		/// </summary>
 		protected class GtkControlConnector : WeakConnector
 		{
+			protected bool buttonPressRetVal = true;
+
 			new GtkControl<TControl, TWidget, TCallback> Handler { get { return (GtkControl<TControl, TWidget, TCallback>)base.Handler; } }
 
 			public void HandleScrollEvent(object o, Gtk.ScrollEventArgs args)
```

Comment:
It might make sense to set this as virtual and override it from `GridConnector`, that way we don't need the extra protected bool for all controls.